host_collection.yml

```diff
@@ -0,0 +1,40 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/host_collection.yml
```

Comment:
I think, you are missing the `test/test_playbooks/tasks/host_collection.yml` file.
It should create a host collection with the same name (as the default value) as you are using in the activation key playbook.
Once you have all those files, you need to record the tests for host_collection and activation_key.
See: https://github.com/theforeman/foreman-ansible-modules/blob/master/README.md#how-to-test-modules-in-this-repository

Reply:
Yup, I'm moving all the test files to their correct folder. Fixtures, playbooks and the missing task file.